### ISSUE TITLE
Sphinx 2.4.0 fails on including svg files due to parsed float dimensions

### ISSUE THREAD

**Issue by hroemer** [sphinx-doc/sphinx#7120]
**Describe the bug**
After updating to Sphinx 2.4.0 my HTML builds start failing due to ```imagesize.get()``` returning float values.

```
Exception occurred:
  File "/home/user/.cache/pypoetry/virtualenvs/project-TWo7gGTJ-py3.8/lib/python3.8/site-packages/sphinx/writers/html5.py", line 548, in visit_image
    atts['width'] = int(atts['width']) * scale
ValueError: invalid literal for int() with base 10: '5760.0'
```

**To Reproduce**

Include an SVG figure

```rest
.. figure:: ./img/some_svg.*
   :scale: 100
```

The error is raised at [html5.py:#L548](https://github.com/sphinx-doc/sphinx/blob/c86e17d4be336286e31e8252b13b9f85d5a7b750/sphinx/writers/html5.py#L548) and due to the returned float values at [images.py#L45](https://github.com/sphinx-doc/sphinx/blob/c86e17d4be336286e31e8252b13b9f85d5a7b750/sphinx/util/images.py#L45).

**Expected behavior**
Image dimensions are converted to ``int``, e.g. like:

```python3
size = imagesize.get(filename)
if size[0] == -1:
    size = None
elif isinstance(size[0], float) or isinstance(size[1], float):
    size = (int(size[0]), int(size[1]))
```

**Environment info**
- OS: [Manjaro 18.1.5 Juhraya]
- Python version: [3.8.1]
- Sphinx version: [2.4.0]

**Comment by tk0miya** [comment]
Sorry for inconvenience. I'll fix this soon.

**Comment by tk0miya** [comment]
Fixed in #7122 
It will be released within a day. Please wait a moment.
